CHANGELOG.md

```diff
@@ -6,6 +6,10 @@
 
 - [`FLAT_DEBUG_ALLOW_HEADER`](/reference/debugging.md#request-debugging) to enable debugging using the `Debug` request header, defaults to `false`
 
+## Fixed
+
+- Empty objects are no longer [logged](/cookbook/custom-logging#adding-a-log-field) as empty arrays.
```

Comment:
Es sollte auch hier erwähnt und in /reference/actions/log.md dokumentiert werden, dass Standardfelder des `flat_access`-Logs nicht in der Log-Action verwendet werden können.